sdk/tests/js/tests/tex-image-and-sub-image-2d-with-image-data.js

```diff
@@ -118,18 +144,20 @@ function generateTest(internalFormat, pixelFormat, pixelType, prologue, resource
 
```

Comment:
This is problematic.  In WebGL, we generate an INVALID_OPERATION if clear* function type and the fbo color image type mismatch.  For example, if you use RED_INTEGER tex, you should use clearBufferiv() instead of clear().